product_docs/docs/sql_assess/1/troubleshooting.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Troubleshooting EDB SQL Assess
+navTitle: Troubleshooting
+---
+
+The EDB SQL Assess tool generates a log file in the same path where the report is generated. The path is provided by the user in the command line options. The tool supports three log levels. If set to `OFF` then log is not generated. If set to `INFO`, which is the default, then all messages are logged except the queries that are assessed. If the level is set to `FINE` then the queries that are assessed are also logged in the file.
+
+To generate reports correctly, EDB SQL Assess needs to resolve the `hostname` to a valid IP address. The easiest way is to add a line like the following in the `/etc/hosts` file:
```

Comment:
@gabbasb What is this note all about?  Why does SQL Assess need to resolve a host?  Isn't everything run locally?

Reply:
@mlewandowski-edb SQL Assess does not need to resolve a host but PLTE does. PLTE connects internally to the H2 database, while connecting it provides a JDBC URL with certain options. We have tried all the possible options but still the JDBC driver needs to resolve hostname. However for most of the operating systems SQLAssess runs without any issues, it is required only in case of SLES.